code/datums/mixed.dm

```diff
@@ -21,6 +21,11 @@
 	size = 5.0
 	var/list/fields = list(  )
 
+/datum/data/record/proc/add_comment(var/comment)
+	var/counter = 1
+	while(fields[text("com_[]", counter)])
```

Comment:
Why are you using `text()`? Just use `[counter]`.

Reply:
it's copypasted from oldcode, I thought I had changed this though